src/components/Footer/index.js

```diff
@@ -1,15 +1,153 @@
 import React from "react";
-import "./styles.scss";
-import { PATHS } from "../../constant";
-import { Link } from "react-router-dom";
+import { Grid, Box, Container, List, Typography, Divider } from "@mui/material";
+import useStyles from "./styles";
+
+const menu = {
+  About: ["Our Story", "Meraki Team "],
+  LearningTracks: [
+    "Python",
+    "Typing Guru",
+    "Spoken English",
+    "Javascript",
+    "Soft Skills",
+    "Open Courses",
+  ],
+  GetInvolved: ["Be a Partner", "Donate", "Careers", "Volunteer"],
+};
+
+const MenuList = (menuItem) => {
+  const title = menuItem.split(/(?=[A-Z])/).join(" ");
+  return (
+    <>
+      <Typography
+        color="text.primary"
+        sx={{ mt: 4 }}
+        variant="subtitle1"
+        component="div"
+      >
+        {title}
+      </Typography>
+      <List>
+        {menu[menuItem].map((item) => (
+          <Typography color="text.primary" sx={{ pb: "5px" }}>
+            {item}
+          </Typography>
+        ))}
+      </List>
+    </>
+  );
+};
 
 function Footer() {
+  const classes = useStyles();
   return (
-    <div className="footer">
-      <Link className="footer-link" to={PATHS.PRIVACY_POLICY}>
-        Privacy Policy
-      </Link>
-    </div>
+    <Box maxWidth="false" bgcolor="primary.light">
+      <Container maxWidth="xl">
+        <Grid container spacing={2} sx={{ mt: "40px" }}>
+          <Grid xs={12} md={4} sx={{ pl: { sm: "none", md: "16px" } }}>
+            <Box sx={{ display: "flex" }}>
+              <Box className={classes.logo}>
+                <img
+                  src={require("./asset/logo.svg")}
+                  alt="social media"
+                  loading="lazy"
+                />
+              </Box>
+              <Box className={classes.meraki}>
+                <img
+                  src={require("./asset/meraki.svg")}
+                  alt="social media"
+                  loading="lazy"
+                />
+              </Box>
+            </Box>
+            <Box sx={{ display: "flex" }}>
+              <Box className={classes.image}>
+                <img
+                  src={require("./asset/facebook.svg")}
+                  alt="social media"
+                  loading="lazy"
+                />
+              </Box>
+              <Box className={classes.image}>
+                <img
+                  src={require("./asset/linkedIn.svg")}
+                  alt="social media"
+                  loading="lazy"
+                />
+              </Box>
+              <Box className={classes.image}>
+                <img
+                  src={require("./asset/twitter.svg")}
+                  alt="social media"
+                  loading="lazy"
+                />
+              </Box>
```

Comment:
You may want to consider making a Component out of footer icons that would look something like this:
```
function FooterIcon(props) {
  return (
    <Box className={classes.image}>
      <img
           src={require("./asset/" + props.name + ".svg")}
           alt={props.alt || "image of " + props.name}
           loading="lazy"
      />
    </Box>
  );
}
```
And then you could rewrite this as something like:
```suggestion
              {['logo', 'meraki', 'facebook', 'linkedIn', 'twitter'].map((imgName) => 
                  <FooterIcon name={imgName} />)}
```

Reply:
I cannot add logo and meraki in that because i gave different class for them